okta/resource_okta_group_memberships.go

```diff
@@ -66,20 +68,16 @@ func resourceGroupMembershipsCreate(ctx context.Context, d *schema.ResourceData,
 }
 
 func resourceGroupMembershipsRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	groupId := d.Get("group_id").(string)
-	users := convertInterfaceToStringSetNullable(d.Get("users"))
-	client := getOktaClientFromMetadata(m)
-	ok, err := checkIfGroupHasUsers(ctx, client, groupId, users)
+	var diags diag.Diagnostics
+
+	logger(m).Info("reading group membership", "id", d.Id())
+	userIDList, err := listGroupUserIDs(ctx, m, d.Id())
```

Comment:
I'm going to defer to @bogdanprodan-okta , it seems like removing the `checkIfGroupHasUsers(` call changes the behavior even though that is the justification from the PR proposal.